integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseOpenUtils.java

```diff
@@ -65,6 +65,10 @@
   private static final int APPEND_PLUS = 0x8402;
   // ax+ FUSE.open() is never called
 
+  // Fuse flags found in actual workloads
+  // used by filebench to read file
+  private static final int READ_TWO = 0xc002;
```

Comment:
I could not find any online material talk about this libfuse open flag.
But filebench read uses this flag to open file for reading